kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -0,0 +1,68 @@
+package com.kodilla.testing.shape;
+
+import com.kodilla.testing.com.kodilla.testing.shape.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ShapeCollectorTestSuite {
+private static int counter=1;
+
+    @Before
+    public void Before (){
+        System.out.println("Start testu nr " + counter );
+    }
+    @After
+    public void After(){
+        System.out.println("Koniec testu nr " + counter );
+        counter++;
+    }
+
+    @Test
+    public void testAddFigure(){
+//given
```

Comment:
given, when, then comments should be also formatted and start in the same place as code.